### DISCUSSION TITLE
Example of using a custom shader

### DISCUSSION THREAD

**Discussion by tokongs** [rnmapbox/maps#3479] [Ideas]
Hey:) 

We're using custom shaders in our maps, and it would be valuable to have an example in this repo for using them with rnmapbox. Would a contribution for a custom shader example be accepted?

**Comment by mfazekas** [comment]
Yes this is more suited as an example than into the library itself. Sure we'd welcome such a contribution.

You can subclass `RNMBXMapComponent` on iOS/swift and  com.rnmapbox.rnmbx.components.AbstractMapFeature on android/kotlin side. And you should be able to implement a component that's interacts with the native map library without adding code to the library itself.
